addons/Assessments_Navigation_Bar/src/presenter.js

```diff
@@ -881,8 +908,20 @@ function AddonAssessments_Navigation_Bar_create(){
         this.setSections();
     };
 
-    presenter.NavigationManager.prototype.shiftPagesToRight = function () {
-        var shift;
+    presenter.NavigationManager.prototype.shiftPagesToRight = function (shift = 0) {
+        if (shift === 0) {
+            shift = this.getShiftToRight();
+            this.shiftCount++;
+        } else {
+            this.shiftCount = Math.floor((presenter.sections.getActualPageIndex() + 1) / (presenter.configuration.numberOfButtons - 4));
+        }
+
+        this.leftSideIndex += shift;
```

Comment:
Nie rozumiem tego fragmentu. 
Jeżeli shift != 0:
* to dodaj shift do this.leftSideIndex
* jednocześnie nie uwzględniając shift w oblieczniu this.shiftCount